pkg/event/config.go

```diff
@@ -220,7 +220,8 @@ func (conf *Config) UnmarshalXML(d *xml.Decoder, start xml.StartElement) error {
 	}
 
 	if len(parsedConfig.QueueList) == 0 {
-		return errors.New("missing queue configuration(s)")
+		*conf = Config(parsedConfig)
+		return nil
```

Comment:
This `return` prevents LambdaList and TopicList checking.  Fix this by running below `for` only `if len(parsedConfig.QueueList() > 0` and add comment `// Empty queue list means user wants to delete the notification configuration.`

Reply:
done